manifests/plugin/gutterball/config.pp

```diff
@@ -0,0 +1,14 @@
+# gutterball config
+class katello::plugin::gutterball::config(
+    $foreman_user  = 'foreman',
+    $foreman_group = 'foreman',
+    $foreman_plugins_dir = '/etc/foreman/plugins',
+  ){
+
+  file { "${foreman_plugins_dir}/foreman_gutterball_auxiliary.yaml":
```

Comment:
I'd use gutterball.yaml since the plugin is also named gutterball. In the PR I mentioned I will also default to that if you don't specify a config filename.